utils/mediaupload.js

```diff
@@ -33,7 +33,9 @@ export function mediaUpload( {
 		filesSet[ idx ] = value;
 		onFileChange( compact( filesSet ) );
 	};
-	const isAllowedType = ( fileType ) => startsWith( fileType, `${ allowedType }/` );
+	const isAllowedType = ( fileType ) => {
+		return ( allowedType === '*' ) || startsWith( fileType, `${ allowedType }/` );
```

Comment:
Should it rather list all allowed file types by allowing an array format?

Reply:
Yes, I think it should accept arrays, as well as have a more predictable syntax. More on that here: #7155

I opted for a wildcard here as a stop-gap measure, or else we'd have something like `['audio', 'application', 'video', 'text', 'image', 'font']` which I think is more error-prone for consumers.